src/Utils.js

```diff
@@ -0,0 +1,48 @@
+import $ from 'jquery';
+import { proxy, minYear, maxYear, defaultYear } from './config';
+
+function parseHtml(html) {
+  let parser = new DOMParser();
+  return parser.parseFromString(html, 'text/html');
+}
+
+exports.fetchDocument = function(url) {
+  return fetch(url)
+    .then((response) => response.json())
+    .then(html => parseHtml(html))
+}
+
+exports.parseInput = function() {
+  let inputYear = $('#inputYear').val();
+  let errorMess = $('.errorMess');
+  let noQuerryMess = $('.noQueryMess');
+  $('#inputYear').val('');
+
+  if (isNaN(Number(inputYear)) || inputYear.length === 0) {
```

Comment:
Use `parseInt` instead of `Number()`. `NaN` is falsey value so you are good going with:
```js
if (parseInt(inputYear, 10))
```